src/transport/FabricTable.h

```diff
@@ -151,10 +150,13 @@ class DLL_EXPORT FabricInfo
 
     CHIP_ERROR GetOperationalCredentials(MutableByteSpan & credentials)
     {
+        // TODO - Refactor storing and loading of fabric info from persistent storage.
+        //        The op cert array doesn't need to be in RAM except when it's being
+        //        transmitted to peer node during CASE session setup.
         ReturnErrorCodeIf(!AreCredentialsAvailable(), CHIP_ERROR_INCORRECT_STATE);
-        ReturnErrorCodeIf(credentials.size() < mNOCCertLen, CHIP_ERROR_BUFFER_TOO_SMALL);
-        memcpy(credentials.data(), mNOCCert, mNOCCertLen);
-        credentials.reduce_size(mNOCCertLen);
+        ReturnErrorCodeIf(credentials.size() < mOperationalCertsLen, CHIP_ERROR_BUFFER_TOO_SMALL);
+        memcpy(credentials.data(), mOperationalCerts, mOperationalCertsLen);
+        credentials.reduce_size(mOperationalCertsLen);
```

Comment:
Watch-out, this larger size always used here due to loss of assumption of "common ICA between peers" may functionally blow-up on some memory constrained devices. It's not very likely today, but we have to keep an eye on it.

Reply:
It makes sense to me. This part of FabricTable will be optimized for RAM usage. That should take care of some such cases.